### PR TITLE
Post-method for user (issue #154)

### PR DESCRIPTION
**Pull request by AndrePu**
Added ability to save edit of session profile in database

**More detailed:**
#154  Profile editing

### CODE REVIEW

File: Cooper/ClientApp/src.v2/app/services/user.service.ts [hunk 1]
```diff
@@ -15,5 +15,16 @@ export class UserService {
     return response;
   }
 
-
+  postData(user: User) {
```
Comment (horba) [inline Cooper/ClientApp/src.v2/app/services/user.service.ts]: too general naming, it doesn't show what method do

File: Cooper/ClientApp/src.v2/app/layouts/profile/profile.layout.ts [hunk 2]
```diff
@@ -80,8 +80,9 @@ export class ProfileLayoutComponent implements OnInit {
       this.updateProfile();
     }
 
-    updateSessionUserInfo(updatedUser: User) {
-
-      // add update user logic
+    updateSessionUserInfo(updatedUser: User): void {
+      if (this.isSessionProfile) {
```
Comment (horba) [inline Cooper/ClientApp/src.v2/app/layouts/profile/profile.layout.ts]: isSessionProfile - isn't clear

File: Cooper/ClientApp/src.v2/app/services/user.service.ts (hunk 1 above)
Comment (AndrePu) [reply Cooper/ClientApp/src.v2/app/services/user.service.ts]: updateUser(..)?

Comment (horba) [reply Cooper/ClientApp/src.v2/app/services/user.service.ts]: save/update is more obvious, yes

**Pull request merged**